ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Update package lists
+  apt: update_cache=yes
```

Comment:
It would be nice to use args of the module as sub-items in commands e.g. 
<pre>
apt:  
  update_cache: yes
</pre>

Reply:
That's much nicer now, thanks :)